src/sass/partials/global/_variables.scss

```diff
@@ -0,0 +1,8 @@
+$font-stack:    HelveticaNeue, Helvetica, Arial, sans-serif;
```

Comment:
There's a more robust Helvetica Neue stack. Can you please use that? It should be on CSS Tricks. Just leave out the `Helvetica Neue Light` bits